src/modules/sensors/vehicle_gps_position/VehicleGPSPosition.cpp

```diff
@@ -258,11 +290,6 @@ VehicleGPSPosition::blend_gps_data()
 			for (uint8_t i = 0; i < GPS_MAX_RECEIVERS; i++) {
 				if (_gps_state[i].fix_type >= 3 && _gps_state[i].s_variance_m_s > 0.0f) {
 					speed_accuracy_sum_sq += _gps_state[i].s_variance_m_s * _gps_state[i].s_variance_m_s;
-
-				} else {
-					// not all receivers support this metric so set it to zero and don't use it
-					speed_accuracy_sum_sq = 0.0f;
-					break;
```

Comment:
Note that these sections were removed so that blending wouldn't fail just because one GPS doesn't publish one of the blending metrics.  As long as one metric for blending exists, blending should continue.